mongorestore/mongorestore_test.go

```diff
@@ -20,6 +25,7 @@ import (
 	"github.com/mongodb/mongo-tools-common/testutil"
 	. "github.com/smartystreets/goconvey/convey"
 	"go.mongodb.org/mongo-driver/bson"
+	driverOpt "go.mongodb.org/mongo-driver/mongo/options"
```

Comment:
for consistency with what we do elsewhere in mongo-tools and mongo-tools-common, let's import this as `mopt`

Reply:
There are a few other places in mongo-tools that still use `driverOpts` as the name. @varsha804 , while you're changing this one, could you also find/replace the others so we can make it consistent?

Reply:
I actually no longer need this import, but will do, Tim!